### PR TITLE
feat: update product copy [CWM-45]

### PR DESCRIPTION
**Pull request by Hricha-Shandily**
# Pull Request Template

## Description

Please include a summary of the change and issue(s) fixed. Also, mention relevant motivation, context, and any dependencies that this change requires.
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Hricha-Shandily** [comment]
**Important points to take care of before publishing:**

- [ ] I couldn't update the copy for a handful of content pieces (such as Dyte and Slack integration desc) because their locations in various files were confusing. 
- [ ] All the pieces of sidebar texts mention placeholder anchor texts that are yet to be hyperlinked to our docs 2.0. Left that because the Help Center's new custom domain is yet to be set up. 
- [ ] The text on the onboarding screen has been heavily edited. The file, however, didn't show any obvious URLs to be updated. So need some help updating these links. 
- [ ] Ensure the updated copy doesn't majorly affect the associated translations.


**Guidelines used:**

- I have used title cases only for main headings and links to those headings. Rest everything has been converted to sentence case.
- The copy has been written keeping [these guidelines](https://www.notion.so/chatwoot/How-to-write-UX-copy-35ae10eccad94b57bdac6b61bf4a18b2?pvs=4) in mind.
- I have tried to maintain consistency everywhere. If anyone catches any discrepancies, kindly LMK.

**Comment by pranavrajs** [comment]
@Hricha-Shandily Broke this down to authentication pages only. The rest of changes are here. You can create a PR from that branch. https://github.com/chatwoot/chatwoot/tree/chore/update-product-copy-1

File: app/javascript/dashboard/i18n/locale/en/login.json [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
       "LABEL": "Password",
       "PLACEHOLDER": "Password"
     },
     "API": {
-      "SUCCESS_MESSAGE": "Login Successful",
-      "ERROR_MESSAGE": "Could not connect to Woot Server, Please try again later",
-      "UNAUTH": "Username / Password Incorrect. Please try again"
+      "SUCCESS_MESSAGE": "Login successful",
+      "ERROR_MESSAGE": "Could not connect to Woot server. Please try again.",
+      "UNAUTH": "Username / Password incorrect. Please try again."
```
Comment (pranavrajs) [inline app/javascript/dashboard/i18n/locale/en/login.json:15]:
```suggestion
      "UNAUTH": "Username or password is incorrect. Please try again."
```

File: app/javascript/dashboard/i18n/locale/en/resetPassword.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
 {
   "RESET_PASSWORD": {
-    "TITLE": "Reset Password",
+    "TITLE": "Reset password",
     "EMAIL": {
       "LABEL": "Email",
-      "PLACEHOLDER": "Please enter your email",
-      "ERROR": "Please enter a valid email"
+      "PLACEHOLDER": "Please enter your email.",
+      "ERROR": "Please enter a valid email."
```
Comment (pranavrajs) [inline app/javascript/dashboard/i18n/locale/en/resetPassword.json:7]: I'm not sure about this change as this is a text in the input placeholder or the error text next to the input. I checked a couple of SaaS products, there doesn't seem to be a common pattern. @Hricha-Shandily Please share your thought process behind this and check if there is a recommended standard.

File: app/javascript/dashboard/i18n/locale/en/setNewPassword.json [hunk 3]
```diff
@@ -1,19 +1,19 @@
 {
   "SET_NEW_PASSWORD": {
-    "TITLE": "Set New Password",
+    "TITLE": "Set new password",
```
Comment (pranavrajs) [inline app/javascript/dashboard/i18n/locale/en/setNewPassword.json:3]: This is the page title.

File: app/javascript/dashboard/i18n/locale/en/signup.json [hunk 4]
```diff
@@ -10,33 +10,33 @@
...
-      "PLACEHOLDER": "Enter your full name. eg: Bruce Wayne",
-      "ERROR": "Full name is too short"
+      "PLACEHOLDER": "Enter your full name. E.g., Bruce Wayne",
+      "ERROR": "Full name is too short."
     },
     "EMAIL": {
       "LABEL": "Work email",
-      "PLACEHOLDER": "Enter your work email address. eg: bruce@wayne.enterprises",
-      "ERROR": "Please enter a valid work email address"
+      "PLACEHOLDER": "Enter your work email address. E.g., bruce@wayne.enterprises",
```
Comment (pranavrajs) [inline app/javascript/dashboard/i18n/locale/en/signup.json:23]: Enter vs please enter. What's preferred?

**Review by pranavrajs — CHANGES_REQUESTED** [review]
Some comments. We will get this merged after the comments are resolved.

**Comment by pranavrajs** [comment]
@Hricha-Shandily Can you please address the changes as per the discussion? Let us get this out.

File: app/javascript/dashboard/i18n/locale/en/resetPassword.json [hunk 5]
```diff
@@ -1,14 +1,14 @@
 {
   "RESET_PASSWORD": {
-    "TITLE": "Reset Password",
+    "TITLE": "Reset password",
```
[the remark this answers is not in the document]
Comment (Hricha-Shandily) [reply app/javascript/dashboard/i18n/locale/en/resetPassword.json:3]: > This is the page title, shall we revert this change?

If we are following the principles suggested, we can keep the original suggestion. Do you have a preference? Quoting the principle down below again. 👇
image

File: app/javascript/dashboard/i18n/locale/en/signup.json (hunk 4 above)
Comment (Hricha-Shandily) [reply app/javascript/dashboard/i18n/locale/en/signup.json:23]: > Enter vs please enter. What's preferred?

We settled on "please" for errors to sound polite and to skip it at other places. Resolving this.

File: app/javascript/dashboard/i18n/locale/en/signup.json [hunk 6]
```diff
@@ -10,33 +10,33 @@
     },
     "COMPANY_NAME": {
       "LABEL": "Company name",
-      "PLACEHOLDER": "Enter your company name. eg: Wayne Enterprises",
-      "ERROR": "Company name is too short"
+      "PLACEHOLDER": "Enter your company name. E.g., Wayne Enterprises",
```
Comment (Hricha-Shandily) [inline app/javascript/dashboard/i18n/locale/en/signup.json:13]:
```suggestion
      "PLACEHOLDER": "Example Enterprises",
```

File: app/javascript/dashboard/i18n/locale/en/signup.json [hunk 7]
```diff
@@ -10,33 +10,33 @@
     },
     "COMPANY_NAME": {
       "LABEL": "Company name",
-      "PLACEHOLDER": "Enter your company name. eg: Wayne Enterprises",
-      "ERROR": "Company name is too short"
+      "PLACEHOLDER": "Example Enterprises",
```
Comment (Hricha-Shandily) [inline app/javascript/dashboard/i18n/locale/en/signup.json:13]:
```suggestion
      "PLACEHOLDER": "Enter your company name. E.g., Wayne Enterprises",
```

**Pull request merged**